microdonuts/src/main/java/com/otsample/api/resources/CASING-NOTE.txt

```diff
@@ -0,0 +1,7 @@
+CASING NOTE FOR OUR RESOURCES CLASSES
+=====================================
+
+We use the Google's gson library for json handling,
+which by default serialized classes fields based
+on their actual name - so we decided to keep them
+json-friendly, instead of going camel-case.
```

Comment:
let's use https://github.com/joelittlejohn/jsonschema2pojo/issues/327#issue-64389558... I'll do this in a follow-on.